feat(docs): add impersonation guide (#395)
blitz-js/blitzjs.com: app/pages/docs/impersonation.mdx

---
title: How To Impersonate Other Users
sidebar_label: How To Impersonate Other Users
---

> Impersonation, also called personification is often a critical tool for SaaS applications as it allows you to view the application from the perspective of a specific user without knowing their credentials.

Reviewer: ```suggestion
Impersonation is often a critical tool for SaaS applications as it allows admins to view the application from the perspective of a specific user without knowing their credentials.
```

"personification" is something totally different :)
